### DISCUSSION TITLE
Disable Download

### DISCUSSION THREAD

**Discussion by ramadhansaad** [DirectoryLister/DirectoryLister#668] [Help & Support] [answered]
HI. thank you so much for the Great script
I need help for Disable Download file (download this dircory)
how i can do it?

**Comment by ramadhansaad** [comment]
Well, I know the solution. thanks

**Reply by PHLAK** [reply]
Glad you figured this out.

**Comment by PHLAK** [comment, marked as answer]
For anyone landing here in the future use the [`ZIP_DOWNLOADS`](https://docs.directorylister.com/configuration/app-config-reference#zip_downloads) environment variable to control this.

**Comment by aiowebtech** [comment]
odd issue?

I've disabled using the "false" variable but...

some files still want to download as zip on mobile

thanks in advance for a tip here
